client/app/lib/components/table/TanStackTableBuilder/useTanStackTableBuilder.tsx

```diff
@@ -12,7 +11,7 @@ import {
   Row,
   useReactTable,
 } from '@tanstack/react-table';
-import { isEmpty, isString } from 'lodash';
+import { isEmpty } from 'lodash';
```

Comment:
Thanks for removing the unused exports introduced in an earlier PR.

Reply:
you welcome!!